### PR TITLE
Version 1.1.0

### PR DESCRIPTION
**Pull request by jprotin**
New: If upc is empty and ean not empty, ean code is synced in Reverb API in the upc field
Fix: Add a new environment variable PS_ERASE_DB=1 in the env_file for the Docker image of PrestaShop
Fix: Update new version of PrestaShop images in the Dockerfile (PrestaShop 1.6.1.16 and 1.7.2.0)

### CODE REVIEW

File: conf/env/PRESTASHOP.env [hunk 1]
```diff
@@ -16,6 +16,7 @@ PS_FOLDER_INSTALL=installOLD
 ADMIN_MAIL=demo@reverb.com
 ADMIN_PASSWD=123456
 PS_HANDLE_DYNAMIC_DOMAIN=0
+PS_ERASE_DB=1
```
Comment (kylecrum) [inline conf/env/PRESTASHOP.env]: what does this do?

Comment (jprotin) [inline conf/env/PRESTASHOP.env]: It's a new variable to re-install the database. If the value is 0, when we launch launch the Docker container, the system don't install the database.

Comment (kylecrum) [inline conf/env/PRESTASHOP.env]: So, it's always going to re-install the database now?

Comment (jprotin) [inline conf/env/PRESTASHOP.env]: By default Yes, if you don't want re-install the database, you can edit the PRESTASHOP.env : PS_ERASE_DB=0.

Comment (kylecrum) [inline conf/env/PRESTASHOP.env]: I don't think I would want to reinstall the db every time that I start the container. Setting up test data over and over again is not a fun task.

Comment (jprotin) [inline conf/env/PRESTASHOP.env]: If you install the container the first Time, you must to configure this variable at 1, it's better if you want à database linked with prestashop. If you're container is stopped, you update this field at 0 and docker could be restart without data base installation.

**Pull request merged**